test/metabase/dashboard_subscription_test.clj

```diff
@@ -41,7 +41,8 @@
                                            :position 0}
                                           pulse-card)]
                   DashboardCard [_ (merge {:dashboard_id (u/the-id dashboard)
-                                           :card_id (u/the-id card)}
+                                           :row          0
```

Comment:
This is the cause of 
https://github.com/metabase/metabase/actions/runs/4343640011/jobs/7585948716#step:4:620

that test uses 2 macros: `with-dashboard-sub-for-card` and `with-dashboard-fixture-for-dashboard` which both adds dashcard, and it was failing because there are 2 cards with the same row=0.